markdown/blog/2025-april-summary.md

```diff
@@ -0,0 +1,80 @@
+
+<a href='https://www.apideck.com/' target='_blank'>
+<img src='https://github.com/asyncapi/spec/blob/master/assets/apideck.png?raw=true' alt='APIDeck' width='500px' />
+</a>
+
+## Sponsorship Updates
+At the beginning of this year, [GitHub](http://github.com/) sent us over $500 as part of their regular annual payment for the use of open-source projects. It is also confirmation that AsyncAPI is contributing to GitHub's development.
+
+On another note, [RedHat](https://www.redhat.com) has terminated their sponsorship. They have changed their marketing strategy, resulting in a $100 decrease in our monthly budget for the Initiative. 
+If anyone knows someone at RedHat who uses AsyncAPI and has a budget for support, please send me a message to [Lukasz Gornicki](https://asyncapi.slack.com/team/UD698Q5LM) or send an email to [info@asyncapi.io](mailto:info@asyncapi.io). 
+
+## AsyncAPI Governance Board 
+The vote for the new AsyncAPI Governance Board is now open. Part of the change will be to replace the single Executive Director position and have the option of Ambassadors becoming TSC members. 
+
+If you're part of the TSC, [please cast your vote on introducing the governance board discussion](https://github.com/asyncapi/community/pull/1634#issuecomment-2822193087).
+
+## AsyncAPI Conference
+
+We had our first AsyncAPI Conference of the year in Singapore, and it was a great success. Thanks to our speakers for taking the time to come and share their knowledge. For a detailed summary, [please read the AsyncAPI Singapore Conference summary]().
+
+### Call for Speakers
+We are a few weeks away before we close the Call for Proposals for [AsyncAPI Conference in Munich](https://conference.asyncapi.com/venue/Munich) on the **11th of May** and [AsyncAPI Conference in Lagos](https://conference.asyncapi.com/venue/Lagos) on the **17th of May**. Please don't forget to submit those talks before the deadline ends; we look forward to you sharing your knowledge with the rest of the community!
+
+While submitting your proposals, please select the "**AsyncAPI**" option for the track to ensure your submission goes to the correct category.
+
+### Call for Sponsors
+We are still open to financial sponsors for the AsyncAPI Conf 2025. [Please help us by sharing the conference prospectus within your networks](https://drive.google.com/file/d/1LxTFLWkRmFQkHOyrwZBL7yqSO2_XPJNq/view?usp=sharing).
+
+## Spec x Tooling 
+[Godwin Alexander](https://github.com/oviecodes) recently opened a discussion about the future of [Glee](https://github.com/asyncapi/glee), noting that it is no longer being actively maintained. He is seeking input from the community on whether we should continue to invest in Glee. [Please share your thoughts in the Future of Glee discussion](https://github.com/orgs/asyncapi/discussions/1861).
+
+## Onboarding/Training Videos
```

Comment:
I think we have option to embed videos, you should try this for this video, to embed in article

Reply:
AAAH I had forgotten about that, I'll add the embed